dist/index.html

```diff
@@ -8,13 +8,13 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="description" content="Official website for Bananium Labs">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-
+    
     <meta property="og:title" content="Bananium Labs">
     <meta property="og:description" content="Official website for Bananium Labs">
     <meta property="og:type" content="website">
     <meta property="og:url" content="https://bananium.com">
-    <meta property="og:image" content="img/favicon.ico">
-    <meta property="og:site_name" content="Bananium Labs">
+    <meta property="og:image" content="https://bananium.com/img/banner-v5.png">
```

Comment:
Why use https instead of doing just `/img/banner-v5.png`?

Reply:
Looks to me the opengraph spec doesn't allow for relative paths. They also got recommended image size, so we might have to resize the image whenever we're free.

Reply:
ok cool, sounds good. maybe make an issue for that so we don't forget?

you can merge this in now tho